addons/acr/units/CfgUnits.hpp

```diff
@@ -0,0 +1,207 @@
+//Units
+class GVAR(Rifleman): GVAR(Soldier_Base) {
+    scope = 2;
+    faction = QUOTE(ADDON);
+    side = 0;
+    displayName = "Rifleman";
+    uniformClass = "CUP_U_B_CZ_WDL_NoKneepads";
+    uksf_equipment_randomUniforms[] = {
+        "CUP_U_B_CZ_WDL_NoKneepads", 1,
+        "CUP_U_B_CZ_WDL_Kneepads_Gloves", 1
+    };
+    weapons[] = {QGVAR(CUP_arifle_ACRC_blk_556_CUP_acc_FlashlightACE_optic_MRCO_2D),"Put","Throw"};
+    respawnWeapons[] = {QGVAR(CUP_arifle_ACRC_blk_556_CUP_acc_FlashlightACE_optic_MRCO_2D),"Put","Throw"};
+    items[] = {"ACE_packingBandage","ACE_packingBandage","ACE_packingBandage","ACE_packingBandage","ACE_packingBandage","ACE_packingBandage","ACE_packingBandage","ACE_packingBandage","ACE_packingBandage","ACE_packingBandage","ACE_elasticBandage","ACE_elasticBandage","ACE_elasticBandage","ACE_elasticBandage","ACE_elasticBandage","ACE_elasticBandage","ACE_elasticBandage","ACE_elasticBandage","ACE_elasticBandage","ACE_elasticBandage"};
+    respawnItems[] = {"ACE_packingBandage","ACE_packingBandage","ACE_packingBandage","ACE_packingBandage","ACE_packingBandage","ACE_packingBandage","ACE_packingBandage","ACE_packingBandage","ACE_packingBandage","ACE_packingBandage","ACE_elasticBandage","ACE_elasticBandage","ACE_elasticBandage","ACE_elasticBandage","ACE_elasticBandage","ACE_elasticBandage","ACE_elasticBandage","ACE_elasticBandage","ACE_elasticBandage","ACE_elasticBandage"};
+    magazines[] = {"CUP_30Rnd_556x45_Stanag","CUP_30Rnd_556x45_Stanag","CUP_30Rnd_556x45_Stanag","CUP_30Rnd_556x45_Stanag","CUP_30Rnd_556x45_Stanag","CUP_30Rnd_556x45_Stanag","CUP_30Rnd_556x45_Stanag_Tracer_Green","CUP_30Rnd_556x45_Stanag_Tracer_Green","CUP_30Rnd_556x45_Stanag_Tracer_Green","SmokeShell","SmokeShell","SmokeShell","CUP_HandGrenade_RGD5","CUP_HandGrenade_RGD5"};
+    respawnMagazines[] = {"CUP_30Rnd_556x45_Stanag","CUP_30Rnd_556x45_Stanag","CUP_30Rnd_556x45_Stanag","CUP_30Rnd_556x45_Stanag","CUP_30Rnd_556x45_Stanag","CUP_30Rnd_556x45_Stanag","CUP_30Rnd_556x45_Stanag_Tracer_Green","CUP_30Rnd_556x45_Stanag_Tracer_Green","CUP_30Rnd_556x45_Stanag_Tracer_Green","SmokeShell","SmokeShell","SmokeShell","CUP_HandGrenade_RGD5","CUP_HandGrenade_RGD5"};
+    linkedItems[] = {"ItemMap","ItemCompass","CUP_NVG_PVS15_green_WP","CUP_H_CZ_Helmet03","CUP_V_CZ_vest04","G_Bandanna_oli","CUP_NVG_PVS15_green_WP"};
+    respawnLinkedItems[] = {"ItemMap","ItemCompass","CUP_NVG_PVS15_green_WP","CUP_H_CZ_Helmet03","CUP_V_CZ_vest04","G_Bandanna_oli","CUP_NVG_PVS15_green_WP"};
+    backpack = "";
+};
+class GVAR(Squad_Leader): GVAR(Rifleman) {
```

Comment:
Need icons to be specified for special units (leader, officer, MG, AT, medic) using `icon = "iconManLeader"`